app/controllers/organisation/settings_controller.rb

```diff
@@ -43,6 +43,26 @@ def destroy
 		
 	end
 
+	def list_respondents
+		respondent_ids = @organisation.respondents.uniq(&:user_id).map(&:user_id)
+		@respondents = User.where(id: respondent_ids).filter_by(params[:page], filter_params.to_h, sort_params.to_h)
+    respond_to do |format|
+      if request.xhr?
+        format.html { render partial: "organisation/settings/respondents_table" }
+      else
+        format.html
+      end
+    end
+	end
+
+	def destroy_respondents
```

Comment:
@BALAJI-BALA - is it possible to destroy a respondent that has already submitted an answer?

Reply:
@mikevic Yeah bro its possible, should I add condition to allow only delete the non-responded respondent?

Reply:
@BALAJI-BALA - yes please.